arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1712 @@
+				"consumes": [],
+				"produces": ["application/json", "text/json"],
+				"parameters": [{
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ResponseWithContinuation[IEnumerable[ResourceProviderOperationInfo]]"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Devices/IotHubs": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get all IotHubs in a subscription",
+				"description": "Get all IotHubs in a subscription",
+				"operationId": "IotHubResource_GetIotHubsForSubscription",
+				"consumes": [],
+				"produces": ["application/json", "text/json"],
+				"parameters": [{
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ResponseWithContinuation[IEnumerable[IotHubDescription]]"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get all IotHubs in a Resourcegroup",
+				"description": "Get all IotHubs in a Resourcegroup",
+				"operationId": "IotHubResource_GetIotHubsForResourceGroup",
+				"consumes": [],
+				"produces": ["application/json", "text/json"],
+				"parameters": [{
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ResponseWithContinuation[IEnumerable[IotHubDescription]]"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/operationresults/{operationId}": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get the result of a long running operation like create or update iothub",
+				"description": "Get the result of a long running operation like create or update iothub",
+				"operationId": "IotHubResource_GetOperationResult",
```

Comment:
Do you really need this? Clientruntime will do the polling and provide the final result.